.github/etc/dictionary.txt

```diff
@@ -1,2 +1,5 @@
 capricornis
 crispus
+kubeconform
+textlint
```

Comment:
Make sure to remove any API keys or secrets in plain text. These should be stored securely.